job-service/src/main/java/com/hpe/caf/services/job/api/DatabaseHelper.java

```diff
@@ -462,6 +433,41 @@ public void cancelJob(final String partitionId, String jobId) throws Exception {
         }
     }
 
+    /**
+     * Cancels the specified jobs
+     */
+    public int cancelJobs(final String partitionId, String jobIdStartsWith, final List<String> labels, final String filter)
+            throws Exception {
+        int successfulCancellations = 0;
+        final int limit = 100;
```

Comment:
Let's make this configurable through an ENV, just in case we need to turn it in production.